### DISCUSSION TITLE
[Enhancement] [select] select多选时能否支持已选的tag自定义内容？

### DISCUSSION THREAD

**Discussion by moxao** [element-plus/element-plus#13156] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

如果可以给select的tag一个slot，那这个组件就完美了。

### Additional comments

_(empty)_

**Comment by RocChuang** [comment]
我也提了这个，不晓得什么时候能加上

**Comment by kooriookami** [comment]
我记录一下

**Comment by kooriookami** [comment]
Added in 2.5.0
